uaclient/exceptions.py

```diff
@@ -29,6 +29,17 @@ def __init__(self, cfg):
         )
 
 
+class MissingAptURLDirective(UserFacingError):
```

Comment:
Excellent, thanks for the clear breadcrumb for if this breaks on ua-contract backend.